devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -23,100 +23,101 @@ metadata {
 		capability "Light"
 
 		// Generic
-		fingerprint profileId: "0104", deviceId: "0101", inClusters: "0006, 0008", deviceJoinName: "Light" //Generic Dimmable Light
+		fingerprint profileId: "0104", deviceId: "0101", inClusters: "0006, 0008", deviceJoinName: "Generic Dimmable Light"
```

Comment:
This pull request cannot contain other changes than Your fingerprints (looks like the older code was submitted).